Dockerfile-base

```diff
@@ -0,0 +1,21 @@
+FROM nginx:1.14.2
```

Comment:
I went with "least disruptive", and stuck with `nginx` as the base, even though we don't need nginx.  These dockerfiles are _just_ for building the angular stuff and writing out to `dist`.